src/main/java/slimeknights/tconstruct/smeltery/client/GuiTinkerTank.java

```diff
@@ -0,0 +1,154 @@
+package slimeknights.tconstruct.smeltery.client;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.inventory.Container;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraftforge.fluids.FluidStack;
+import slimeknights.mantle.client.gui.GuiElement;
+import slimeknights.mantle.inventory.BaseContainer;
+import slimeknights.tconstruct.common.TinkerNetwork;
+import slimeknights.tconstruct.library.Util;
+import slimeknights.tconstruct.library.client.RenderUtil;
+import slimeknights.tconstruct.library.smeltery.SmelteryTank;
+import slimeknights.tconstruct.smeltery.network.SmelteryFluidClicked;
+import slimeknights.tconstruct.smeltery.tileentity.TileTinkerTank;
+
+public class GuiTinkerTank extends GuiContainer {
+
+  public static final ResourceLocation BACKGROUND = Util.getResource("textures/gui/tinker_tank.png");
+  protected GuiElement scala = new GuiElement(122, 0, 106, 106, 256, 256);
+
+  public TileTinkerTank tinkerTank;
+
+  public GuiTinkerTank(Container inventorySlotsIn, TileTinkerTank tinkerTank) {
+    super(inventorySlotsIn);
+
+    this.xSize = 122;
+    this.ySize = 130;
+
+    this.tinkerTank = tinkerTank;
+  }
+
+  @Override
+  protected void drawGuiContainerForegroundLayer(int mouseX, int mouseY) {
+    // draw the scale
+    this.mc.getTextureManager().bindTexture(BACKGROUND);
+    GlStateManager.color(1.0F, 1.0F, 1.0F, 1.0F);
+    scala.draw(8, 16);
+
+    // draw the tooltips, if any
+    // subtract the corner of the GUI location so the mouse location is relative to just the center, rather than the inventory center
+    mouseX -= guiLeft;
+    mouseY -= guiTop;
+
+    // Liquids
+    if(8 <= mouseX && mouseX < 114 && 16 <= mouseY && mouseY < 122) {
+      FluidStack hovered = getFluidHovered(122 - mouseY - 1);
+      List<String> text = Lists.newArrayList();
+
+      if(hovered == null) {
```

Comment:
It was a little hard to move to a superclass since the smeltery uses on GuiMultiModule (which requires an inventory).

I could dump this stuff in render utils.